testsuite/smoke-test/src/rosetta.rs

```diff
@@ -94,18 +178,32 @@ async fn test_block() {
         genesis_block.block_identifier,
         genesis_block.parent_block_identifier
     );
+    assert_eq!(
+        HashValue::zero().to_hex(),
+        genesis_block.block_identifier.hash,
+    );
+    assert_eq!(0, genesis_block.block_identifier.index);
 
-    // Genesis timestamp is always 0
-    assert_eq!(0, genesis_block.timestamp);
+    // Genesis timestamp is always Y2K
+    assert_eq!(Y2K_SECS, genesis_block.timestamp);
```

Comment:
In your rosetta implementation do you overwrite the genesis block timestamp to be this value?

Reply:
Yes, it always comes out as this value, though it's always 0 onchain.